governance.md

```diff
@@ -82,7 +82,7 @@ Once you reach the consensus:
 
 ### Lack of consensus
 
-Engage the ultimate decision makers if there are still unresolved objections by the decision due date. The ultimate decision makers are stated in the decision log of the **Decision record** issue. Work with them to reach the final decision as soon as possible and close the issue.
+Engage [the Kyma Council](https://github.com/kyma-project/community/tree/master/sig-and-wg#the-kyma-council), which is the ultimate decision maker, if there are still unresolved objections by the decision due date. Work with them to reach the final decision as soon as possible and close the issue.
```

Comment:
There is a call to action for engaging Kyma council but it doesn't say how. If I were someone from the community I would expect that it will lead to a slack channel or an email address to the Kyma council.

Reply:
I'll update that once the communication channel to Kyma Council is set up

Reply:
@agolawska 
 please check this blog post 
https://wiki.hybris.com/display/ysf/2018/10/19/Decisions%2C+notifications+and%2C+the+Kyma+Council

Reply:
@natasab we decided yesterday that we would prefer that people could contact with SIG Core leads in case of lack of consensus